src/content/docs/en/reference/components-reference.mdx

```diff
@@ -0,0 +1,263 @@
+---
+title: Built-in Components Reference
+i18nReady: true
+---
+import Since from '~/components/Since.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import ReadMore from '~/components/ReadMore.astro';
+
+## Built-in Components
+
+Astro includes several built-in components for you to use in your projects. All built-in components are available in `.astro` files via `import {} from 'astro:components';`.
+
+You can reference the `Props` of these components using the [`ComponentProp` type](/en/guides/typescript/#componentprops-type) utility.
+
+### `<Code />`
+
+```astro 'theme="dark-plus"' /wrap\b/ /(inline) \/>/
```

Comment:
This has nothing to do with the split but as I was rereading I was wondering... Shouldn't `defaultColor` also be highlighted? (if the goal is to highlight the attributes cited in line 36)
````suggestion
```astro 'theme="dark-plus"' /wrap\b/ /(inline) 'defaultColor={false}' \/>/
````

Note: I'm not used to Expressive Code, so feel free to double-check that I'm using the correct syntax. I took over the one from `theme`...